cmake/SanitizerOptions.cmake

```diff
@@ -0,0 +1,13 @@
+if(ENABLE_SANITIZER)
+  message(STATUS "Enabling address sanitizer (ASan).")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address")
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fsanitize=address")
+  set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -fsanitize=address")
+endif()
+
+if(ENABLE_UB_SANITIZER)
```

Comment:
For consistent naming, this flag should be `ENABLE_UNDEFINED_SANITIZER`.

Reply:
Done